lib/strong_interface/parameters_validator.rb

```diff
@@ -33,8 +33,10 @@ def valid?
       end
     end
 
-    def description(metod)
-      metod.inspect.scan(/[.|#](\S+?\(.*?\))/).last.first
+    private
+
+    def description(method)
+      method.inspect.scan(/[.|#](\S+?\(.*?\))/).last.first
```

Comment:
I prefer not to use the name of one of the internal methods as a name of a local variable :)

Could you please revert this change, or suggest another name for this variable?

Reply:
Wow, I thought that was a typo.

I use something like `mthd` or `meth` for such cases, however I have reverted that change. This is ok :ok_hand: